src/Transition/CreateNewIDFUsingRulesV8_6_0.f90

```diff
@@ -718,6 +718,17 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                   OutArgs(1)='*'
                   nodiff=.false.
                 ENDIF
+
+                 ! For Output:Variable that reference a specific Daylighting:Controls object need update to the new reference point name
+                 IF (InArgs(1) .NE. '*') THEN
+                   IF (SameString(InArgs(2)(1:29),'Daylighting Reference Point 1')) THEN
+                     OutArgs(1) = TRIM(InArgs(1)) // '_DaylRefPt1'
+                   ENDIF
+                   IF (SameString(InArgs(2)(1:29),'Daylighting Reference Point 2')) THEN
+                     OutArgs(1) = TRIM(InArgs(1)) // '_DaylRefPt2'
+                   ENDIF
```

Comment:
And the transition rules file needs to be updated to include these rules.  This will conflict with #5864 so the rules file will need a manual merge either here or there.